web/modules/custom/portland/modules/portland_address_verifier/js/addressVerifierModel.js

```diff
@@ -189,7 +206,30 @@ AddressVerifierModel.prototype.callSecondaryQuery = function (url, x, y, callbac
     });
 }
 
-AddressVerifierModel.getPropertyByPath = function (obj, path, parse = "stringify", omit_nulls = false) {
+AddressVerifierModel.getPropertyByPath = function (jsonObject, path) {
```

Comment:
My new version of getPropertyByPath wasn't backwards compatible, so I restored the old version and added a new version. This can be removed once the 2 forms that use the legacy secondary query functionality are reconfigured to use the new array format.